estate/models/house_type.py

```diff
@@ -0,0 +1,6 @@
+from odoo import fields, models
+
+class house_type(models.Model):
+    _name = 'estate.house_type'
```

Comment:
```suggestion
    _name = 'estate.house.type'
```
The used convention in odoo is using words seperated by dots.
Using an underscore is not wrong and it's used in some cases but it's not common